pkg/cmd/openshift-controller-manager/controller/interfaces.go

```diff
@@ -37,16 +37,19 @@ type ControllerContext struct {
 	// ClientBuilder will provide a client for this controller to use
 	ClientBuilder ControllerClientBuilder
 
-	ExternalKubeInformers   kexternalinformers.SharedInformerFactory
-	AppInformers            appinformer.SharedInformerFactory
-	BuildInformers          buildinformer.SharedInformerFactory
-	ImageInformers          imageinformer.SharedInformerFactory
-	NetworkInformers        networkinformer.SharedInformerFactory
-	TemplateInformers       templateinformer.SharedInformerFactory
-	QuotaInformers          quotainformer.SharedInformerFactory
-	AuthorizationInformers  authorizationinformer.SharedInformerFactory
-	RouteInformers          routeinformer.SharedInformerFactory
-	SecurityInformers       securityinformer.SharedInformerFactory
+	ExternalKubeInformers kexternalinformers.SharedInformerFactory
+
+	InternalBuildInformers         buildinformer.SharedInformerFactory
```

Comment:
prefixing the imports with internal would make it easy to check that you are actually making internal variable of internal type

Reply:
@tnozicka i can do that, but it will make the diff bigger and harder to review ;-) can do as a follow up...